src/user/model.coffee

```diff
@@ -11,9 +11,12 @@ BLANK_USER =
   name: null
   profile_url: null
   courses: []
+  _course_data: []
+  endpoints: {}
+  isLoaded: false
```

Comment:
I mean that we send out a `status/auth` on open, like here: https://github.com/openstax/concept-coach/blob/master/src/concept-coach/base.cjsx#L64 . This gets called on mount at which point since the user is logged out, the `status/auth` called will be made.  But I guess it doesn't hurt to not reset it, so I'll change it back.